src/app/components/elements/panels/QuestionFinderFilterPanel.tsx

```diff
@@ -158,7 +158,10 @@ export function QuestionFinderFilterPanel(props: QuestionFinderFilterPanelProps)
     const deviceSize = useDeviceSize();
     const dispatch = useAppDispatch();
     const pageContext = useAppSelector(selectors.pageContext.context);
-    const bookOptions = ISAAC_BOOKS.filter(book => !pageContext?.subject || book.subject === pageContext?.subject);
+    const bookOptions = ISAAC_BOOKS.filter(b => !b.hidden).filter(book => 
+        (!pageContext?.subject || book.subject === pageContext?.subject)
+        && (!pageContext?.stage || book.stages.some(x => pageContext?.stage?.includes(x)))
```

Comment:
```suggestion
        && (!pageContext?.stage || !(pageContext?.stage.length > 0) || book.stages.some(x => pageContext?.stage?.includes(x)))
```
Similarly to concepts - this is hiding the books filter on the generic QF.